js/README.md

```diff
@@ -122,7 +122,7 @@ const rainfall = Table.new(
 ```js
 import { Table } from "apache-arrow";
 
-const table = await Table.from(fetch(("/simple.arrow")));
+const table = await Table.from(fetch("/simple.arrow"));
```

Comment:
where can I download a copy of this `/simple.arrow` file?

Reply:
Not sure but that's beyond the scope of this fix.